INSTALL.md

```diff
@@ -1,16 +1,35 @@
 # Install
 
+## Quick Start
+
+If you just want to try out the astrobee simulator, you can use one of the docker images in the [Docker Hub](https://hub.docker.com/r/astrobee/astrobee). Make sure you have docker installed in your ubuntu system flolowing the [installation instructions](https://docs.docker.com/engine/install/ubuntu/) and [post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/).
+Given that Docker is installed, just run:
```

Comment:
If this is a sequence of steps the user should do, formatting it as a numbered list would make it easier to follow. Maybe make explicit checking out the main astrobee repo.

Reply:
And I guess make explicit the proper 'docker pull' command as well. How do you pick which docker image to use? Just trying to think of places the user could get lost.

Reply:
I added instructions similar to the install instructions format